src/js/server/attendance/email-templates.js

```diff
@@ -23,4 +23,18 @@ module.exports = {
       },
     };
   },
+  expiry({ name, daysValid }) {
+    return {
+      subject: 'Your Hack Cambridge invitation has expired',
+      body: {
+        name,
+        intro: [
+          `Earlier we sent you an invitation to Hack Cambridge Recurse with ${daysValid} days to respond. We have not received a response from you and your invitation has expired.`,
+          'We hope to see you apply for the next Hack Cambridge!'
```

Comment:
I agree. All of the emails say this and it's hurt me a bit every time I've written one. There _is_ a link in the footer which is slightly more tolerable.